manifests/c/CoreyButler/NVMforWindows/1.1.9/CoreyButler.NVMforWindows.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: CoreyButler.NVMforWindows
+PackageVersion: 1.1.9
+PackageLocale: en-US
+Publisher: Corey Butler
```

```suggestion
Publisher: Ecor Ventures LLC
```
This must match Control Panel.